core/authz.go

```diff
@@ -61,4 +62,25 @@ var policyByRoute = map[string][]string{
 
 	"/raft/join": {"internal"},
 	"/raft/msg":  {"internal"},
+
+	"/":                                       {"public"},
+	"/dashboard":                              {"public"},
+	"/dashboard/core":                         {"public"},
```

Comment:
Seeing all these makes me think as a todo perhaps we should allow matches like the way ServeMux works, so we could write `/dashboard/` here and get all of them.